1-exercises/A-undefined/exercise.js

```diff
@@ -11,7 +13,7 @@
 
 // Example 1
 let a;
-console.log(a);
+console.log(a);// 'a' has already been declared
```

Comment:
'a' hasn't been assigned any value, it is just an empty variable, that's why it's coming up as undefined.